File: entity/src/main/java/com/zhuangjy/bean/Job.java
package com.zhuangjy.bean;

/**
 * Created by zhuangjy on 2016/1/8.
 */
public class Job {
    String jobName;
    String companyCity;
    String companyName;
    Float workYear;
    Float salary;
    String education;
    String financeStage;
    String industryField;
    Float companySize;


    public Job(String jobName, String companyCity, String companyName, Float workYear, Float salary, String education, String financeStage, String industryField, Float companySize) {
        this.jobName = jobName;
        this.companyCity = companyCity;
        this.companyName = companyName;
        this.workYear = workYear;
        this.salary = salary;
        this.education = education;
        this.financeStage = financeStage;
        this.industryField = industryField;
        this.companySize = companySize;
    }

    public String getJobName() {
        return jobName;
    }

    public void setJobName(String jobName) {
        this.jobName = jobName;
    }

    public String getCompanyCity() {
        return companyCity;
    }

    public void setCompanyCity(String companyCity) {
        this.companyCity = companyCity;
    }

    public String getCompanyName() {
        return companyName;
    }

    public void setCompanyName(String companyName) {
        this.companyName = companyName;
    }

    public Float getWorkYear() {
        return workYear;
    }

    public void setWorkYear(Float workYear) {
        this.workYear = workYear;
    }

    public Float getSalary() {
        return salary;
    }

    public void setSalary(Float salary) {
        this.salary = salary;
    }

    public String getEducation() {
        return education;
    }

    public void setEducation(String education) {
        this.education = education;
    }

    public String getFinanceStage() {
        return financeStage;
    }

    public void setFinanceStage(String financeStage) {
        this.financeStage = financeStage;
    }

    public String getIndustryField() {
        return industryField;
    }

    public void setIndustryField(String industryField) {
        this.industryField = industryField;
    }

    public Float getCompanySize() {
        return companySize;
    }

    public void setCompanySize(Float companySize) {
        this.companySize = companySize;
    }

    @Override
    public String toString(){
        return jobName + " " + companyCity + " " + companyName + " " + workYear + " " + salary + " " + education + " " +
                financeStage + " " + industryField + " " + companySize + "\r\n";
    }
}

File: data-analysis/src/main/java/com/zhuangjy/bean/Job.scala
package com.zhuangjy.bean

/**
  * Created by zhuangjy on 2016/2/17.
  */
class Job (var _jobId:Long,var _jobName:String,var _jobType:String,var _companyCity:String,var _companyName:String,
           var _workYear:Float,var _salary:Float,var _education:String,var _financeStage:String,var _industryField:String,
           var _companySize:Float){
  @scala.reflect.BeanProperty
  var jobId = _jobId
  @scala.reflect.BeanProperty
  var jobName = _jobName
  @scala.reflect.BeanProperty
  var jobType = _jobType
  @scala.reflect.BeanProperty
  var companyCity = _companyCity
  @scala.reflect.BeanProperty
  var companyName = _companyName
  @scala.reflect.BeanProperty
  var workYear = _workYear
  @scala.reflect.BeanProperty
  var salary = _salary
  @scala.reflect.BeanProperty
  var education = _education
  @scala.reflect.BeanProperty
  var financeStage = _financeStage
  @scala.reflect.BeanProperty
  var industryField = _industryField
  @scala.reflect.BeanProperty
  var companySize = _companySize
}
